connect-examples/v2/node_bookings/.npmrc

```diff
@@ -1 +1,2 @@
 engine-strict=true
+registry=http://registry.npmjs.org/
```

Comment:
It seems dangerous to bypass TLS like this. I just tried `cd connect-examples/v2/node_bookings && npm i` on my machine without `registry=http://registry.npmjs.org/` in this file, and it succeeded. @azlee @barakj are you able to npm install without this registry override too?

Reply:
I think this is mainly to prevent internal square developers from locking the dependencies to the Square artifactory.

Reply:
Could we set it to the https registry url in that case?
```
registry=https://registry.npmjs.org/
```